src/ints/bios.cpp

```diff
@@ -1032,17 +1032,117 @@ void BIOS_ZeroExtendedSize(bool in) {
 	if(other_memsystems < 0) other_memsystems=0;
 }
 
+static void shutdown_tandy_sb_dac_callbacks()
+{
+	/* abort DAC playing */
+	if (tandy_sb.port) {
+		IO_Write(tandy_sb.port + 0xc, 0xd3);
+		IO_Write(tandy_sb.port + 0xc, 0xd0);
+	}
+	real_writeb(0x40, 0xd4, 0x00);
+	if (tandy_DAC_callback[0]) {
+		uint32_t orig_vector = real_readd(0x40, 0xd6);
+		if (orig_vector == tandy_DAC_callback[0]->Get_RealPointer()) {
+			/* set IRQ vector to old value */
+			uint8_t tandy_irq = 7;
+			if (tandy_sb.port) {
+				tandy_irq = tandy_sb.irq;
+			} else if (tandy_dac.port) {
+				tandy_irq = tandy_dac.irq;
+			}
+			uint8_t tandy_irq_vector = tandy_irq;
+			if (tandy_irq_vector < 8) {
+				tandy_irq_vector += 8;
+			} else {
+				tandy_irq_vector += (0x70 - 8);
+			}
+
+			RealSetVec(tandy_irq_vector, real_readd(0x40, 0xd6));
+			real_writed(0x40, 0xd6, 0x00000000);
+		}
+		delete tandy_DAC_callback[0];
+		delete tandy_DAC_callback[1];
+		tandy_DAC_callback[0] = nullptr;
+		tandy_DAC_callback[1] = nullptr;
+	}
+	tandy_sb.port  = 0;
+	tandy_dac.port = 0;
+}
+
+void BIOS_SetupTandySbDacCallbacks()
+{
+	/* tandy DAC can be requested in tandy_sound.cpp by initializing this
+	 * field */
```

Comment:
Minor: maybe just use `//` comments everywhere?

Reply:
Switched them over (the code here is existing and just relocated verbatim)